scripts/task_process/cache_status.py

```diff
@@ -180,14 +180,11 @@ def parseJobLog(fp, nodes, node_map):
         while len(info['WallDurations']) > len(info['SiteHistory']):
             info['SiteHistory'].append("Unknown")
 
-def parseErrorReport(fp, nodes):
-    def last(joberrors):
-        return joberrors[max(joberrors, key=int)]
-    data = json.load(fp)
+def parseErrorReport(data, nodes):
     #iterate over the jobs and set the error dict for those which are failed
     for jobid, statedict in nodes.iteritems():
         if 'State' in statedict and statedict['State'] == 'failed' and jobid in data:
-            statedict['Error'] = last(data[jobid]) #data[jobid] contains all retries. take the last one
+            statedict['Error'] = data[jobid].values()[0] #data[jobid] contains all retries. take the last one
```

Comment:
Why do we need `values()[0]` and can't just put a string with the error if there is only one thing?